spec/rubocop/config_loader_spec.rb

```diff
@@ -286,6 +286,32 @@
         expect(configuration_from_file['Style/StringLiterals']['Include']).to eq(['dir/**/*.rb'])
       end
 
+      it 'does not expand a nil Include' do
+        create_file(file_path, <<~YAML)
+          inherit_from: ../.rubocop.yml
+          Style/StringLiterals:
+            Include: ~
+        YAML
+
+        expect(configuration_from_file['Style/StringLiterals']['Include']).to be_nil
+      end
+
+      it 'gets an Include that is relative to the subdirectory when inheriting from a nil Include' do
```

Comment:
Not so sure about this test. I wanted to mostly check that going from not nil to nil and vice versa worked